packages/cli/src/server.ts

```diff
@@ -24,6 +24,15 @@ program
   .description('Launch OpenAPI server')
   .option('-p, --port <number>', 'Optionally set port to override config')
   .action(async (cmd) => {
+    const { version } = createObjects(getConfig(program.opts().config), {
+      version: '/version',
+    })
+
+    if (version !== 1) {
```

Comment:
I think it should also check if a version is specified:
```suggestion
    if (version && version !== 1) {
```

and perhaps be lenient with the type with `!=` instead of `!==`?